### PR TITLE
convert templates to greenbar

### LINKED ISSUE #1202: Update groove bundle templates to Greenbar
We're dropping support for bundle configuration v3 with Cog 0.17, and thus need to update the [groove](https://github.com/cogcmd/groove) bundle's templates from Mustache to Greenbar.

### PR DESCRIPTION
**Pull request by mpeck**
resolves https://github.com/operable/cog/issues/1202

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -68,26 +68,31 @@ commands:
...
+      *From:* ~$result.ticket.customer_name~
+      *Subject:* ~$result.ticket.title~
+      *Date:* ~$result.ticket.created_at~
 
-      {{#messages}}
-      > _*{{from}}*_
-      >{{{slack_body}}}
-
-      {{/messages}}
+      ~if cond=$result.messages no_empty?~
```
Comment (vanstee) [inline config.yaml]: Should that be `not_empty?`?

File: templates/slack/ticket.greenbar [hunk 2]
```diff
@@ -0,0 +1,11 @@
+~each var=$results as=result~
+*From:* ~$result.ticket.customer_name~
+*Subject:* ~$result.ticket.title~
+*Date:* ~$result.ticket.created_at~
+
+~if cond=$result.messages no_empty?~
```
Comment (vanstee) [inline templates/slack/ticket.greenbar]: ditto `not_empty?` from above

File: config.yaml (hunk 1 above)
Comment (mpeck) [reply config.yaml]: oops, you are correct.

File: config.yaml [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ---
-cog_bundle_version: 3
+cog_bundle_version: 4
 name: groove
 description: Groove helpdesk commands
 version: 0.2.0
```
Comment (vanstee) [inline config.yaml]: Probably need to bump this as well.

**Review by vanstee — APPROVED** [review]

**Pull request closed without merging**